src/docs/self-hosted/index.mdx

```diff
@@ -27,7 +27,11 @@ Environment="HTTPS_PROXY=http://proxy:3128"
 Environment="NO_PROXY=127.0.0.0/8"
 ```
 3. Run `systemctl daemon-reload` and restart Docker with `systemctl restart docker.service`.
-4. To add your proxy environment variables into Sentry's Docker containers, create `~/.docker/config.json` with these contents:
+
+From there you can run `./install.sh` like usual.
+
+By default Sentry sends anonymous usage statistics to the Sentry team. It helps to improve the product. To make this feature work it's required to allow Docker containers to access the Internet. To add your proxy environment variables into Sentry's Docker containers, create `~/.docker/config.json` with these contents:
```

Comment:
From this sentence it seems like sentry containers only want to access the Internet for "anonymous usage statistics", is it so?

Let's say if users want to integrate their sentry with a third party service like slack, I think they need to use proxy there as well.

This should be added to the sentence too IMHO.

Reply:
fair enough